x-pack/plugins/enterprise_search/public/applications/app_search/components/document_creation/document_creation_buttons.tsx

```diff
@@ -34,11 +33,7 @@ interface Props {
 export const DocumentCreationButtons: React.FC<Props> = ({ disabled = false }) => {
   const { openDocumentCreation } = useActions(DocumentCreationLogic);
 
-  const { engineName, isSampleEngine } = useValues(EngineLogic);
-  const {
-    myRole: { canViewEngineCrawler },
-  } = useValues(AppLogic);
-  const showCrawlerLink = canViewEngineCrawler && !isSampleEngine;
```

Comment:
I also just confirmed in the Crawler channel w/ Rich that there's no realistic scenario where a user would have access to document creation (upload/pasting JSON) and not have access to the Crawler, so I think this access check is also safe to remove